### DISCUSSION TITLE
Gender translation

### DISCUSSION THREAD

**Discussion by emmexx** [filamentphp/filament#8767] [Help]
In languages different from english nouns can be masculine and feminine. And the associated adjectives follow the gender of the noun.
In Filament when you create a new record the header of the create page is `New Name-of-the-model`.
In other languages the translation of New depends on the gender of the model word.
Example in italian:
Model name Indirizzo (address) -> header title Nuov**o** Indirizzo
Model name Stanza (room) -> header title Nuov**a** Stanza

I didn't find anything to manage this. And I suppose there are other contests where this problem arises.
I found a "Create **a** Photo to begin" that translates to italian "Crea un foto per iniziare" but foto is feminine and the correct translation should be "Crea un**a** foto per iniziare".

One simple solution is to change the way **new**, **a** and other key words that can have a genre in other languages with a gender neutral form (in italian Nuovo/a, un/a).

Is there any other way already implemented for other languages?

**Comment by Efois** [comment]
How could I help you in this?

**Comment by weppami** [comment]
Have you come up with a workaround for this ? I'm in the same situation

**Comment by ahmic** [comment]
You can define a custom title for Resource pages like this

```
\\ App\Filament\Resources\AddressResource\Pages\CreateAddress.php

...

public function getTitle(): string | Htmlable
{
    return __('Your custom page title');
}

...

```
